javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,15 @@
+/* exported getStudentNames */
+// function getStudentNames(students) {
+//   var newArr = students.map(person => person.name);
+//   return newArr;
+// }
+
+function getStudentNames(students) {
+  var newArr = [];
+  for (var i = 0; i < students.length; i++) {
+    for (var name in students[i]) {
```

Comment:
This `for in` loop is unnecessary, you already know the name of the property you want to push.

Reply:
If there were multiple properties on the object, you would be storing every property value, not just `name`.